contrib/custom_templates.go

```diff
@@ -54,3 +56,25 @@ var ClientProviders = func() model.CustomTemplates {
 
 	return clientProvidersTemplate
 }()
+
+/*
+InputSnapshot custom template
+*/
+const (
+	InputSnapshotCustomTemplatePath = "input/input_snapshot.gotmpl"
+)
+
+var InputSnapshot = func(outputFilename, groupModule string, selectFromGroups []model.Group, resourcesToSelect map[schema.GroupVersion][]string) model.CustomTemplates {
```

Comment:
Since this is the top-level function users will be interacting with, can we include more extensive documentation here as well.